src/technologies/s.json

```diff
@@ -2569,8 +2570,7 @@
     ],
     "saas": true,
     "scriptSrc": [
-      "sdks\\.shopifycdn\\.com",
-      "cdn\\.shopify\\.com"
+      "sdks\\.shopifycdn\\.com"
     ],
```

Comment:
@rockeynebhwani can you remove this `scriptSrc` completely?

And also add the `powered-by` to lines 225?

```json
    "headers": {,
      "powered-by": "Shopify"
      "x-shopid": "\\;confidence:50",
      "x-shopify-stage": ""
    },
```